circle.yml

```diff
@@ -15,4 +15,5 @@ test:
         - docker run -t civisanalytics/datascience-python /bin/bash -c "python -c 'import numpy'"
         - docker run civisanalytics/datascience-python python -c "from numpy.distutils import system_info; assert system_info.get_info('mkl') == {}"
         - docker run civisanalytics/datascience-python python -c "import numpy; numpy.test()"
-        - docker run civisanalytics/datascience-python /bin/bash -c "conda list|grep -c conda-forge|python -c \"import sys; assert int(sys.stdin.readlines()[0]) == 5, 'There should be 5 conda-forge packages'\""
+        - docker run civisanalytics/datascience-python /bin/bash -c "conda list|grep conda-forge"
```

Comment:
For our purposes, it seems a little tricky to tell which packages (and for which exact versions) have just been added to conda-forge in order to make sure the conda-forge package count assertion test will pass at any given point of time. So I've added this line just to show what the build thinks the conda-forge packages are -- useful when the assertion test fails and we have to change the expected package count just one line below.

Reply:
It's also useful even if the assertion test passes. We know exactly what the conda-forge packages are:

```
arrow-cpp                 0.5.0               np113py36_2    conda-forge
certifi                   2017.7.27.1              py36_0    conda-forge
jemalloc                  4.4.0                         0    conda-forge
parquet-cpp               1.2.0                         0    conda-forge
pyarrow                   0.5.0               np113py36_0    conda-forge
requests                  2.18.4                   py36_1    conda-forge
xgboost                   0.6a2                    py36_2    conda-forge
```